src/Resource/Iterator/ResourceCollectionIterator.php

```diff
@@ -165,7 +165,7 @@ public function next()
     public function key()
     {
         if (null === ($key = key($this->resources))) {
-            return null;
+            return;
```

Comment:
Could you undo this change? Since `key()` returns a value, `return null;` should be preferred.